bazel-mode.el

```diff
@@ -73,6 +73,18 @@
   (when bazel-mode-buildifier-before-save
     (bazel-mode-buildifier)))
 
+(defconst bazel-mode--builtins
```

Comment:
nit: I think it's fine to inline these two constants. They are only used once, and their meaning is clear given their usage position.

Reply:
Done.